### PR TITLE
Fix WebSdk packages

### PR DESCRIPTION
**Pull request by mmitche**
The packages contained multiple copies of Sdk.props and Sdk.targets, as well as some others (analyzers). This was because the Content itemgroup was being filled multiple times. Once in each project file, and once when the final layout of the package was laid out.

### CODE REVIEW

**Comment by vijayrkn** [comment]
Thanks @mmitche . This only impacts the final NuGet packages right? I have checked the SDKs folder and the files do get added properly & the package by itself is not used any where. It is still good to keep the package clean even though it can't be consumed as a package 👍

**Comment by mmitche** [comment]
> Thanks @mmitche . This only impacts the final NuGet packages right? I have checked the SDKs folder and the files do get added properly & the package by itself is not used any where. It is still good to keep the package clean even though it can't be consumed as a package 👍

None of these packages are shipping. This package fails validation later on becuase you can't unpack it without overwriting existing files (because it has two Sdk.props and Sdk.targets files with the same name and location)

File: src/WebSdk/Publish/Tasks/Microsoft.NET.Sdk.Publish.Tasks.csproj [hunk 1]
```diff
@@ -49,15 +49,15 @@
       <DependentUpon>Resources.resx</DependentUpon>
     </Compile>
     
-    <Content Include="$(PublishRoot)\Targets\**\*.*">
+    <AdditionalContent Update="$(PublishRoot)\Targets\**\*.*">
```
Comment (vijayrkn) [inline src/WebSdk/Publish/Tasks/Microsoft.NET.Sdk.Publish.Tasks.csproj:52]: Include to be consistent?

Comment (mmitche) [reply src/WebSdk/Publish/Tasks/Microsoft.NET.Sdk.Publish.Tasks.csproj:52]: Ahh I messed that up, let me fixup.

**Review by vijayrkn — APPROVED** [review]

**Review by vijayrkn — APPROVED** [review]

**Pull request merged**